### DISCUSSION TITLE
Can't connect to planetscale via SQLProd Studio

### DISCUSSION THREAD

**Discussion by pmsteil** [planetscale/discussion#397] [Q&A] [answered]
Hello, I have been able to connect to planetscale via MySqlWorkbench, but not SQLPro Studio - weird.

The key to solving this is probably on this screen:
https://share.cleanshot.com/dkZ6mJTG

I have tried it as it is, setting the CA Certificate to my /etc/ssl/cert.pem (I am on MacOS) and it won't connect... 

Any other tips?

**Comment by jgreet** [comment]
Do you get a SSL related error back from SQL Pro Studio?

One option could be using [`pscale connect`](https://github.com/planetscale/cli) to proxy the connection through a local port with SSL disabled in the client.

https://planetscale.com/docs/tutorials/connect-any-application#option-2-connect-using-the-planetscale-proxy

**Comment by fooforge** [comment, marked as answer]
Hey @pmsteil,

I have experimented with SQLPro Studio today and could not get it to connect to a PlanetScale database either. We often see SSL/TLS related issues, but I have spotted a successful SSL handshake in the application's logs so it's fairly safe to say that this isn't the issue. My gut says this is more Vitess-related.

I have written it all up for the developer of SQLPro Studio. Chime in and/or subscribe to https://github.com/hankinsoft/SQLPro/issues/884.

**Reply by fooforge** [reply]
Looks like the developer's issued a new release that allows you to connect to PlanetScale now. Marking as solved. :-)

**Reply by pmsteil** [reply]
Yep, works for me also! :)
